modules/dxp/apps/commerce/commerce-machine-learning-api/src/main/java/com/liferay/commerce/machine/learning/forecast/SkuCommerceMLForecastManager.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.machine.learning.forecast;
+
+import aQute.bnd.annotation.ProviderType;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Riccardo Ferrari
+ */
+@ProviderType
+public interface SkuCommerceMLForecastManager {
+
+	public SkuCommerceMLForecast addSkuCommerceMLForecast(
+			SkuCommerceMLForecast skuCommerceMLForecast)
+		throws PortalException;
+
+	public SkuCommerceMLForecast create();
+
+	public List<SkuCommerceMLForecast> getMonthlyQuantitySkuCommerceMLForecasts(
+			Date actualDate, long companyId, int end, int forecastLength,
+			int historyLength, String sku, int start)
+		throws PortalException;
```

Comment:
Hey @riccardo-ferrari I asked bchan about this. He said we can go either way with sorting it as alphabetical or logical. Most of our serviceImpls have it logical right now. I think we should keep it that way. He also mentioned that for json, he would sort by alpha.

```suggestion
	public List<SkuCommerceMLForecast> getMonthlyQuantitySkuCommerceMLForecasts(
			Date actualDate, long companyId, int forecastLength,
			int historyLength, String sku, int start, int end)
		throws PortalException;
```

Reply:
Thank you @dsitu , greatly appreciated